test/helpers/types/types.go

```diff
@@ -54,6 +59,14 @@ type Collector struct {
 	Version    string    `json:"version"`
 }
 
+// EventData encodes an eventrouter event and previous event, with a verb for
+// whether the event is created or updated.
+type EventData struct {
```

Comment:
Do we need to define "Event" here in lieu of something that already formally exists in Kubernetes?

Reply:
"EventData" is EventRouter citizen, with Kubernetes Event inside